ImagePicker/LayoutConfiguration.swift

```diff
@@ -25,6 +25,8 @@ public struct LayoutConfiguration {
     /// Spacing between camera section and assets section
     public var cameraSectionSpacing: CGFloat = 10
     
+    public var enableAssetDragAndDrop = false
```

Comment:
should also be available only on ios 11+

Reply:
Stored properties cannot be marked potentially unavailable with '@available'